.github/workflows/stale-issue.yml

```diff
@@ -91,3 +91,31 @@ jobs:
             If you are still interested in contributing this, please ensure that
             it is rebased against the latest branch (usually `master`), all review
             comments have been addressed and the build is passing.
+  close-pending-reproducer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          # GLOBAL ------------------------------------------------------------
+          operations-per-run: 30
+          exempt-all-milestones: true
+          days-before-stale: -1
+          only-labels: 'pending:reproducer'
+
+          # ISSUES ------------------------------------------------------------
+          stale-issue-label: 'pending:reproducer'
+          close-issue-label: 'closed:unreproducible'
+          close-issue-message: >
+            This issue has been automatically closed due to inactivity. If you can reproduce this on a
+            recent version of Gradle or if you have a good use case for this feature, please feel free
+            to let know so we can reopen the issue. Please try to provide steps to reproduce, a quick
+            explanation of your use case or a high-quality pull request.
+
+          # PULL REQUESTS -----------------------------------------------------
+          stale-pr-label: 'pending:reproducer'
+          close-pr-label: 'closed:not-fixed'
```

Comment:
This makes no sense IMO.
It would be better to have no message and only `days-before-pr-close: -1` as a clear indication that this bot action should never close a PR.

From other places in the file, it looks like the message might be mandatory, then consider using the one above highlighting that the closing is a bug.

Reply:
I think could be left unset, should we also remove the other unused close messages?

Reply:
Let's leave it like that for now.